### ISSUE TITLE
Draft.js Equation inline style doesn't update new state when clear equation inline ?

### ISSUE THREAD

**Issue by thanhtungdp** [ory-am/editor#40]
Demo code: https://inyono.github.io/editor/
When add new equation inline style, it's work well.
But I do delete (typing Backspace), it's not clear equation.
<img width="606" alt="screen shot 2016-06-11 at 1 14 14 am" src="https://cloud.githubusercontent.com/assets/4653102/15974284/de7a5d84-2f71-11e6-991b-8a6366d4bd8d.png">

**Comment by inyono** [comment]
Yes, deleting currently behaves pretty weird when we there is some inline equations. This was just a small demo to find out what's feasible with Draft.js but it definitely needs more optimization.

We currently do some prototypes involving layout building (see #35 for a discussion) and will come back later to Draft.js (which is our current favorite for editing text in those layout blocks). So I currently have no solution for this problem. Feel free to comment here if you find a solution yourself. I will update this issue when we work at Draft.js again.
